AzSentinel/Public/Import-AzSentinelAlertRule.ps1

```diff
@@ -157,16 +157,17 @@ function Import-AzSentinelAlertRule {
                 $uri = "$script:baseUri/providers/Microsoft.SecurityInsights/alertRules/$($guid)?api-version=2019-01-01-preview"
             }
 
+            # The official API schema indicates that the grouping configuration is part of the incident configuration
             try {
                 $groupingConfiguration = [GroupingConfiguration]::new(
```

Comment:
maybe add an extra if statement for supporting the older templates?

Reply:
Good catch, @pkhabazi :) Fixed and tested!

I also noticed that I had implemented a breaking change that would prohibit fusion, mlbehavior and MS incident creation rules from being imported, that is fixed as well.